tools/make/kube.mk

```diff
@@ -43,7 +43,7 @@ kube-undeploy: kube-uninstall ## Uninstall the Envoy Gateway controller into the
 	$(tools/kustomize) build internal/provider/kubernetes/config/default | kubectl delete --ignore-not-found=$(ignore-not-found) -f - 
 
 .PHONY: run-kube-local ## Run EG locally.
-run-kube-local: kube-install
+run-kube-local: build kube-install
```

Comment:
while you are here can you also move `## Run EG locally.` to this line, so it shows up in `make help`, TIA

Reply:
Done.